include/language.php

```diff
@@ -639,14 +619,25 @@ public function get_locale( $filter = 'raw' ) {
 	}
 
 	/**
-	 * Returns the values of this instance's properties.
+	 * Returns the values of this instance's properties, which can be filtered if required.
 	 *
 	 * @since 3.4
 	 *
-	 * @return array
+	 * @param bool $raw Whether or not properties should be raw. Default to `true`.
+	 *
+	 * @return array Array of language object properties.
+	 *
+	 * @phpstan-return LanguageData
 	 */
-	public function get_object_vars() {
-		return get_object_vars( $this );
+	public function get_object_vars( $raw = true ) {
```

Comment:
I suppose the raw values should be used for the transient and the filtered values everywhere else.
So I think we should set `$raw` to `false` by default.

Reply:
Ouch! Thanks.

Reply:
Done in dfa485d.